pkg/csr/csr.go

```diff
@@ -305,7 +305,7 @@ func matchesHostname(nodeName string, windowsInstances []*instance.Info,
 
 // findHostName returns the actual host name of the instance by running the 'hostname' command
 func findHostName(instanceInfo *instance.Info, instanceSigner ssh.Signer) (string, error) {
-	win, err := windows.New("", "", instanceInfo, instanceSigner)
+	win, err := windows.New("", "", instanceInfo, instanceSigner, "")
```

Comment:
Is it worth a comment here as to why the `platformType` parameter is empty, something along the lines of `We don't need to pass the platform here because ...`?

Reply:
Done